### PR TITLE
Create mini app

### PR DESCRIPTION
**Pull request by FridaTveit**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,79 @@
-# DevOps-Module-07-Workshop
\ No newline at end of file
+# Installs
+
+- [Visual Studio Professional 2019](https://visualstudio.microsoft.com/downloads/)
+    - Make sure you have the [NPM Task Runner](https://marketplace.visualstudio.com/items?itemName=MadsKristensen.NPMTaskRunner) extension
```
Comment (Rob-Owen) [inline README.md:4]: One of the challenges with this course if getting learners set up with dev tooling. I suspect VS Pro is simply a no-go for most learner cohorts, and even VS Community would be very difficult (we've had issues getting even git pre-authorised by some employers).

We also need to support both Windows and Mac dev envirorments (ideally Linux too!). Today I learned that VSIX extensions are Windows-only! :(

For these reasons, I suggest you adjust the project to run in VS Code, and remove the Visual Studio configuration and docs. I'm not particularly up-to-date with the .NET world, but I think it should be doable with an aspnet core app!

DevOps learners are all already using VS Code, and it'll be an interesting learning point to set up the same editor for C# (they've been doing Python / Docker stuff so far).

File: DotnetTemplate.Web/Views/Shared/_Layout.cshtml [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+  <main class="nhsuk-main-wrapper" id="maincontent">
+    @RenderBody()
+  </main>
+</div>
+
+<footer role="contentinfo">
+  <div class="nhsuk-footer" id="nhsuk-footer">
+    <div class="nhsuk-width-container">
+      <h2 class="nhsuk-u-visually-hidden">Support links</h2>
+      <p class="nhsuk-footer__copyright">&copy; Softwire @DateTime.Now.Year</p>
```
[the remark this answers is not in the document]
Comment (Rob-Owen) [reply DotnetTemplate.Web/Views/Shared/_Layout.cshtml:103]: This is probably a question for @aeone

**Review by Rob-Owen — COMMENTED** [review]
Thanks @FridaTveit - the mini app looks like a great starting point for a workshop.

I've left a couple of comments in-line - the main one being about dev tooling. In short, can you switch Visual Studio for VS Code in the editor config and README?

File: README.md [hunk 3]
```diff
@@ -1 +1,79 @@
...
+# Running the app
+
+The project should now build. Confirm this via *Build* → *Build Solution* (or `CTRL+SHIFT+B`).
+
+You can now run the app by clicking the play button (▶), which should say *IIS Express*.
+
+This should launch the website at: [https://localhost:44363/](https://localhost:44363/)
+
+# Running the tests
+These tests will also be run by the Jenkins job whenever you push.
```
[the remark this answers is not in the document]
Comment (FridaTveit) [reply README.md:27]: Ah yes, this is just left over from the app I adapted it from, will remove

File: DotnetTemplate.Web/Views/Shared/_Layout.cshtml (hunk 2 above)
[the remark this answers is not in the document]
Comment (FridaTveit) [reply DotnetTemplate.Web/Views/Shared/_Layout.cshtml:103]: I didn't think it mattered really as we're not publishing it? All the UI is pretty random though so I'm happy to remove this bit if you prefer.

[the remark this answers is not in the document]
Comment (Rob-Owen) [reply DotnetTemplate.Web/Views/Shared/_Layout.cshtml:103]: I agree it doesn't feel important, I just don't know if Corndel feel the same!

File: README.md (hunk 1 above)
Comment (FridaTveit) [reply README.md:4]: Okay I can try to make it work in VS Code

**Review by thehereward — APPROVED** [review]

**Pull request merged**